src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -113,8 +113,10 @@ def __init__(self, pt_module, graph_element=None, example_input=None, freeze=Tru
                     " yourself, please refer to PyTorch documentation: "
                     "https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html.")
             self.graph_element = pt_module.inlined_graph
+            self.alias_db = self.graph_element.alias_db()
```

Comment:
does it always have `alias_db()` method for both `trace` and `script` cases?

Reply:
Yes, it does